packages/esm-patient-programs-app/src/programs/programs-detailed-summary.component.tsx

```diff
@@ -100,21 +106,21 @@ const ProgramsDetailedSummary: React.FC<ProgramsDetailedSummaryProps> = ({ patie
               renderIcon={(props) => <Add size={16} {...props} />}
               iconDescription="Add programs"
               onClick={launchProgramsForm}
-              disabled={availablePrograms?.length && eligiblePrograms?.length === 0}
+              disabled={availablePrograms?.length && eligiblePrograms?.length === 0 && noCompletedPrograms}
```

Comment:
```suggestion
```

I don't think we should worry about disabling the button because you shouldn't be able to see it if the `isEnrolledInAllPrograms` condition is truthy anyway:

![CleanShot 2024-07-10 at 12  04 26@2x](https://github.com/openmrs/openmrs-esm-patient-chart/assets/8509731/6db50262-06bf-4bf8-a074-8bf2f61e07ad)